include-fragment-element.js

```diff
@@ -1,5 +1,3 @@
-/* eslint-disable github/no-then */
-
```

Comment:
🤔 these are not caught anymore?

Reply:
I thought we may had dropped them but now I'm not sure 🤔 I'll look into this.

Reply:
Oh they are no longer in the `es6` config an are now in the `app` config. See: https://github.com/github/eslint-plugin-github/pull/34